app/controllers/exports_controller.rb

```diff
@@ -32,7 +32,7 @@ def create
     @export.sites = search.to_global_id.to_s
 
     respond_to do |format|
-      if @export.save
+      if @export.save and signed_in?
```

Comment:
don't test for signed_in? user here, we'll need to look at setting up cancancan for doing this sort of thing